.github/workflows/load-tests.yml

```
@@ -6,6 +6,10 @@ on:
       - "v[0-9]+.[0-9]+.[0-9]+*"
     paths-ignore:
       - "**/README.md"
+  pull_request:
+    types: [labeled]
```

```suggestion
    types: [opened, synchronize, reopened, labeled, unlabeled]
```
We'd want this to be the same functionality as [running windows tests with the label `Run Windows`.](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/95ff5f2a611d3adde1f74779f0b7ba374af137c7/.github/workflows/build-and-test-windows.yml#L11)